### PR TITLE
Implemented password reset functionality

### PR DESCRIPTION
**Pull request by siddharthnandy**
Updated screens: 
`mobile/Screens/ForgotPasswordScreen.js`

New routes:
`/passwordReset/create`: requires an `email` param
`/passwordReset/delete`: requires `email` and `passwordResetCode` params, sends back `id`
`/passwordReset/reset`: requires `id` and `password` param

### CODE REVIEW

File: mobile/Screen/ForgotPasswordScreen.js [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+import React, { useState, useRef } from 'react';
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
 import axios from 'axios';
 import { URL } from '@env';
 import {
-  StyleSheet, Text, TextInput, View, Button, Alert, TouchableOpacity, Dimensions,
+  StyleSheet, Text, TextInput, View, Image, Alert, TouchableOpacity, Pressable,
 } from 'react-native';
 import { login } from '../redux/sliceAuth';
```
Comment (he-is-james) [inline mobile/Screen/ForgotPasswordScreen.js:9]: `login` and `useDispatch` are not used, please remove unused variables

File: mobile/Screen/ForgotPasswordScreen.js [hunk 2]
```diff
@@ -65,18 +147,252 @@ function ForgotPasswordScreen({ navigation }) {
...
+    return false;
+  };
+
+  const handleSendCode = async () => {
+    try {
+      const res = await axios.patch(`${URL}/passwordReset/create`, { email });
+      console.log(res);
+    } catch (err) {
+      console.error(err);
+      return err;
```
Comment (he-is-james) [inline mobile/Screen/ForgotPasswordScreen.js:195]: You need a return in the `try` block as well or remove the return `err`

File: mobile/Screen/ForgotPasswordScreen.js [hunk 3]
```diff
@@ -65,18 +147,252 @@ function ForgotPasswordScreen({ navigation }) {
...
+      return true;
+    } catch (err) {
+      console.error(err);
+      return err;
+    }
+  };
+
+  const handleResetPassword = async () => {
+    try {
+      const res = await axios.patch(`${URL}/passwordReset/reset`, { id, password });
```
Comment (he-is-james) [inline mobile/Screen/ForgotPasswordScreen.js:217]: remove `res` if it's never used

**Review by he-is-james — APPROVED** [review]
Looks great! Just a few small improvements needed

File: mobile/Screen/ForgotPasswordScreen.js (hunk 2 above)
Comment (siddharthnandy) [reply mobile/Screen/ForgotPasswordScreen.js:195]: made it return a bool because some page-setting functionality depends on it

**Pull request merged**